build.bash

```diff
@@ -258,11 +258,11 @@ elif [ "$1" == "local-test" ]; then
   # Use local filesystem's version of openHABian
   # shellcheck disable=SC2016
   if ! running_in_docker; then
-    sed -i 's|$(eval "$(openhabian_update &> /dev/null)") -eq 0|true|' /boot/first-boot.bash
+    sed -i 's|$(eval "$(openhabian_update "${clonebranch:-openHAB3}" &> /dev/null)") -eq 0|true|' /boot/first-boot.bash
```

Comment:
Should this not have been in already?

But what does this have to do with CI changes?

Reply:
I noticed the bug while working on ci changes and this particular function is called in CI so I fixed it.